asar-pi-applications/asar_search_algorithm/terrain.txt

```diff
@@ -0,0 +1 @@
+{"coordinate": [[8, 8], [8, 4], [8, 3], [8, 2], [8, 7], [8, 6], [8, 5], [7, 8], [7, 7], [7, 6], [7, 5], [7, 2], [7, 4], [7, 3], [7, 1], [6, 8], [6, 7], [6, 1], [6, 6], [6, 5], [6, 4], [6, 3], [6, 2], [5, 7], [5, 3], [5, 2], [5, 1], [5, 8], [5, 6], [5, 5], [5, 4], [4, 4], [4, 2], [4, 1], [4, 8], [4, 7], [4, 6], [4, 5], [4, 3], [3, 4], [3, 2], [3, 1], [3, 8], [3, 7], [3, 6], [3, 5], [3, 3], [2, 4], [2, 3], [2, 2], [2, 1], [2, 8], [2, 7], [2, 6], [2, 5], [1, 4], [1, 3], [1, 2], [1, 1], [1, 8], [1, 7], [1, 6], [1, 5], [8, 1]], "color": ["gray", "gray", "gray", "gray", "gray", "gray", "gray", "blue", "blue", "blue", "blue", "green", "gray", "green", "green", "blue", "blue", "red", "gray", "gray", "gray", "green", "green", "blue", "gray", "red", "red", "green", "gray", "green", "green", "red", "red", "red", "green", "gray", "green", "green", "gray", "red", "red", "red", "green", "gray", "green", "red", "gray", "gray", "red", "red", "red", "gray", "green", "gray", "gray", "red", "red", "red", "red", "purple", "gray", "gray", "red", "white"]}
```

Comment:
Yes, it's a sample json file. I mentioned this in the pull request comment when I made it because I was not sure whether or not it should be included.